packages/cli/src/commands/domains/index.ts

```diff
@@ -150,6 +151,32 @@ www.example.com  CNAME            www.example.herokudns.com
     return columnHeaders.map(header => headerToKeyMap[header.trim()] || header.trim())
   }
 
+  outputCSV = (customDomains: Heroku.Domain[], tableConfig: Record<string, any>, sortProperty?: string) => {
+    const getValue = (domain: Heroku.Domain, key: string, config?: Record<string, any>) => {
+      const cfg = config ?? tableConfig[key]
```

Comment:
Can we make `cfg` a little more descriptive? Like `tableConfigValue` or `columnValue`? This will help us maintain readability.